src/components/ContextMenu/ContextMenuItem.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import ContextMenu from '.';
+
+function ContextMenuItem({ item, parentRef, isToLeft }) {
```

Comment:
Where we don't use TypeScript, we love to use [React PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html) which helps us be consistent, catch mistakes and share what each component expects.

**Don't** add it here as the project doesn't use it right now, but read up if you want.

Reply:
Will check this out! 🧑‍🏭